src/CSModule.sol

```diff
@@ -106,11 +111,11 @@ contract CSModule is
         _disableInitializers();
     }
 
-    /// @notice initialize the module
+    /// @notice initialize the module from scratch
     function initialize(
         address _accounting,
         address admin
-    ) external initializer {
+    ) external reinitializer(2) {
```

Comment:
I see no changes in the method body. Do we need to change the modifier here?

Reply:
If we want to have versioning we need to have right contract version after deploy from scratch